10_nwis_pull/src/nwis_combine_functions.R

```diff
@@ -26,6 +26,8 @@ combine_nwis_data <- function(ind_file, ...){
 
 convert_to_long <- function(flow_dat) {
   
+  if(nrow(flow_dat) == 0) return(NULL)
```

Comment:
Oops - meant for this change to be committed separately from everything else. This is the single line that pertains to #29